web/versioned_docs/version-0.13.0/auth/social-auth/discord.md

```diff
@@ -0,0 +1,549 @@
+          <div>{children}</div>
+        </div>
+      </div>
+    </div>
+  )
+}
+```
+
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```tsx title="src/pages/auth.tsx"
+import { LoginForm } from 'wasp/client/auth'
+
+export function Login() {
+  return (
+    <Layout>
+      <LoginForm />
+    </Layout>
+  )
+}
+
+// A layout component to center the content
+export function Layout({ children }: { children: React.ReactNode }) {
+  return (
+    <div className="w-full h-full bg-white">
+      <div className="min-w-full min-h-[75vh] flex items-center justify-center">
+        <div className="w-full h-full max-w-sm p-5 bg-white">
+          <div>{children}</div>
+        </div>
+      </div>
+    </div>
+  )
+}
+```
+
+</TabItem>
+</Tabs>
+
+We imported the generated Auth UI components and used them in our pages. Read more about the Auth UI components [here](../../auth/ui).
+
+### Conclusion
+
+Yay, we've successfully set up Discord Auth! 🎉
+
+![Discord Auth](/img/auth/github.png) <!-- TODO: Grab screenshot of discord auth -->
+
+
+Running `wasp db migrate-dev` and `wasp start` should now give you a working app with authentication.
+To see how to protect specific pages (i.e., hide them from non-authenticated users), read the docs on [using auth](../../auth/overview).
+
+## Default Behaviour
+
+Add `discord: {}` to the `auth.methods` dictionary to use it with default settings.
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```wasp title=main.wasp
+app myApp {
+  wasp: {
+    version: "^0.13.0"
+  },
+  title: "My App",
+  auth: {
+    userEntity: User,
+    methods: {
+      // highlight-next-line
+      discord: {}
+    },
+    onAuthFailedRedirectTo: "/login"
+  },
+}
+```
+
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```wasp title=main.wasp
+app myApp {
+  wasp: {
+    version: "^0.13.0"
+  },
+  title: "My App",
+  auth: {
+    userEntity: User,
+    methods: {
+      // highlight-next-line
+      discord: {}
+    },
+    onAuthFailedRedirectTo: "/login"
+  },
+}
+```
+
+</TabItem>
+</Tabs>
+
+<DefaultBehaviour />
+
+## Overrides
+
+<OverrideIntro />
+
+### Data Received From Discord
+
+We are using Discord's API and its `/users/@me` endpoint to get the user data.
+
+The data we receive from Discord on the `/users/@me` endpoint looks something this:
+
+```json
+{
+  "id": "80351110224678912",
+  "username": "Nelly",
+  "discriminator": "1337",
+  "avatar": "8342729096ea3675442027381ff50dfe",
+  "verified": true,
+  "flags": 64,
+  "banner": "06c16474723fe537c283b8efa61a30c8",
+  "accent_color": 16711680,
+  "premium_type": 1,
+  "public_flags": 64,
+  "avatar_decoration_data": {
+    "sku_id": "1144058844004233369",
+    "asset": "a_fed43ab12698df65902ba06727e20c0e"
+  }
+}
+```
+
+The fields you receive will depend on the scopes you requested. The default scope is set to `identify` only. If you want to get the email, you need to specify the `email` scope in the `configFn` function.
```

Comment:
You say here the default scope it `identity` but in the code we put `identity` and `email`?